src/components/visualizations/chart/highcharts/test/getOptionalStackingConfiguration.spec.ts

```diff
@@ -104,18 +104,14 @@ describe("getOptionalStackingConfiguration", () => {
             },
         );
 
-        it.each([
-            ["normal", { stackMeasures: true }, { stackMeasuresToPercent: false }],
-            ["percent", { stackMeasuresToPercent: true }, { stackMeasuresToPercent: true }],
-        ])(
+        it.each([["normal", { stackMeasures: true }], ["percent", { stackMeasuresToPercent: true }]])(
```

Comment:
We have the constants for `percent` and `normal`, right?

Reply:
done